frontend/packages/dev-console/src/components/monitoring/overview/MonitoringTab.tsx

```diff
@@ -1,13 +1,28 @@
 import * as React from 'react';
 import { OverviewItem } from '@console/shared';
-import MonitoringOverview from './MonitoringOverview';
+import { Firehose } from '@console/internal/components/utils';
+import MonitoringOverviewWrapper from './MonitoringOverviewWrapper';
 
 type MonitoringTabProps = {
   item: OverviewItem;
 };
 
-const MonitoringTab: React.FC<MonitoringTabProps> = ({ item: { obj: res, events } }) => {
-  return <MonitoringOverview resource={res} events={events} />;
+const MonitoringTab: React.FC<MonitoringTabProps> = ({ item }) => {
+  const { obj } = item;
+  const resources = [
+    {
+      isList: true,
+      kind: 'Event',
+      namespace: obj?.metadata.namespace,
+      prop: 'events',
+    },
+  ];
```

Comment:
Can we use a field selector to only fetch events related to specific resource.

Looking at `events.jsx`, this is what they are doing. They also don't use firehose. I'm not familiar enough with the history behind why it's done differently here.

`involvedObject.uid=${uid},involvedObject.name=${name},involvedObject.kind=${kind}`

Reply:
Added the selector but stuck with Firehose